### PR TITLE
feat(node-error): CHK-2719 update node error mocks

### PR DESCRIPTION
**Pull request by infantesimone**
#### List of Changes
- update node error mock for ecommerce flow
- partial disable pagopa-proxy mock (as deprecated flow)

#### Motivation and Context
The goal of this PR is to update mock node error handler according new ( @MarcoTormen :-) ) requirement, described in this [docs](https://pagopa.atlassian.net/wiki/spaces/I/pages/867565570/eCommerce+-+Errori+in+verifica+attivazione+e+autorizzazione).

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/handlers/payments.ts [hunk 1]
```diff
@@ -1,639 +1,643 @@
...
-import { Pay3ds2UsingPOSTT } from "../generated/payment_manager/requestTypes";
-import { logger } from "../logger";
-import { PayRequest } from "../generated/payment_manager/PayRequest";
-import { GatewayFaultEnum } from "../generated/pagopa_proxy/GatewayFault";
-import { PartyConfigurationFaultEnum } from "../generated/pagopa_proxy/PartyConfigurationFault";
-import { PartyTimeoutFaultEnum } from "../generated/pagopa_proxy/PartyTimeoutFault";
-import { PaymentStatusFaultEnum } from "../generated/pagopa_proxy/PaymentStatusFault";
-import { ValidationFaultEnum } from "../generated/pagopa_proxy/ValidationFault";
-import { ActivationState, store } from "../store/store";
+/**
```
Comment (pietro-tota) [inline src/handlers/payments.ts:1]: why not delete this file completely? can always be restored from git history, if needed

**Review by pietro-tota — APPROVED** [review]
Sounds good

File: src/flow.ts [hunk 2]
```diff
@@ -85,23 +85,23 @@ export enum SendPaymentResultOutcomeCase {
 export enum FlowCase {
   OK,
   OK_ENABLE_PERSISTENCE,
-  /* pagopa-proxy: getPaymentInfo */
+  /* getPaymentInfo */
   ANSWER_VERIFY_NO_ENTE_BENEFICIARIO,
-  FAIL_VERIFY_400_PPT_STAZIONE_INT_PA_SCONOSCIUTA,
-  FAIL_VERIFY_404_PPT_DOMINIO_SCONOSCIUTO,
+  FAIL_VERIFY_400_INVALID_INPUT,
```
Comment (pietro-tota) [inline src/flow.ts:90]: Are this change compatible with Checkout f.e. CR pipeline integration tests compatible?

File: src/handlers/payments.ts (hunk 1 above)
Comment (infantesimone) [reply src/handlers/payments.ts:1]: I can't do it, _"troppi ricordi"_

Comment (pietro-tota) [reply src/handlers/payments.ts:1]: image

File: src/flow.ts (hunk 2 above)
Comment (infantesimone) [reply src/flow.ts:90]: https://github.com/pagopa/pagopa-checkout-fe/pull/329

File: src/app.ts [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 import { addWalletHandler, updateWalletHandler } from "./handlers/wallet";
 import {
   checkTransactionHandler,
   resume3ds2Handler
 } from "./handlers/transactions";
 import { getPspListHandler } from "./handlers/psps";
-import { ID_PAYMENT, SESSION_USER, USER_DATA } from "./constants";
-import { logger } from "./logger";
+import { ID_PAYMENT, SESSION_USER } from "./constants";
+// import { logger } from "./logger";
```
[the remark this answers is not in the document]
Comment (infantesimone) [reply src/app.ts:23]: related to _"troppi ricordi"_

**Review by ciuffagianluca — APPROVED** [review]

**Pull request merged**